maxim/logger/components/utils.py

```
@@ -0,0 +1,81 @@
+import base64
+import re
+from typing import TYPE_CHECKING, Any, Tuple
```

Ruff (0.11.9)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)